packages/components/src/date-time/date-time/index.tsx

```diff
@@ -37,6 +38,23 @@ function UnforwardedDateTimePicker(
 	}: DateTimePickerProps,
 	ref: ForwardedRef< any >
 ) {
+	if ( ! __nextRemoveHelpButton ) {
+		deprecated( 'Help button in wp.components.DateTimePicker', {
+			since: '13.4',
+			version: '14.6', // Six months of plugin releases.
+			hint:
+				'Set the `__nextRemoveHelpButton` prop to `true` to remove this warning and opt in to the new behaviour, which will become the default in a future version.',
+		} );
+	}
+	if ( ! __nextRemoveResetButton ) {
+		deprecated( 'Reset button in wp.components.DateTimePicker', {
+			since: '13.4',
+			version: '14.6', // Six months of plugin releases.
```

Comment:
Interesting. I believe that I always used the WordPress version (and not the Gutenberg plugin version) when working on components. Would be interesting to see which approach is the expected one!

Reply:
I've always seen and used the Gutenberg version... it's an interesting question because the code ends up both in the plugin and Core... hmmm! 🤔

Reply:
At least the version numbers in Gutenberg are _way_ ahead of core, so it's easy to tell them apart 😁